source/features/no-modals.tsx

```diff
@@ -12,8 +12,8 @@ function init(signal: AbortSignal): void {
 	delegate([
 		'a[href$="/issues/new/choose"]', // New issue button
 		'a[class*="SubIssueTitle"]', // Sub-issue links
-		'a[class^="TitleHeader-module__inline"]', // Global Issue title
-		'[class*="base-cell-module"] a', // Project issue links
+		'a[data-testid="issue-pr-title-link"]', // Global Issue title
```

Comment:
Does the title refer to the global issue list? "Global issue list links" would be more appropriate